### PR TITLE
add indonesian translation

### PR DESCRIPTION
**Pull request by kiraware**
Support Bahasa Indonesia translation

### CODE REVIEW

**Comment by kiraware** [comment]
@ababic  Ready for review from maintainer.

File: CONTRIBUTORS.md
Comment (MrCordeiro) [inline CONTRIBUTORS.md:1]: Please only add your name to contributors, don't change the formatting style for the listed items.

**Review by MrCordeiro — CHANGES_REQUESTED** [review]
Hi @kiraware, thanks for your contribution!

Please restore the formatting style of the CONTRIBUTORS.md files and you're good to go

File: CONTRIBUTORS.md
Comment (kiraware) [reply CONTRIBUTORS.md:1]: I'm sorry, it happen accidentally

Comment (MrCordeiro) [reply CONTRIBUTORS.md:1]: No worries , we've all been there 😅

**Comment by kiraware** [comment]
@MrCordeiro Done.

**Review by MrCordeiro — APPROVED** [review]
Hi, @kiraware , thanks for your translation !

On the CONTRIBUTORS.md file, please only add your name. Don't change the formatting style for the listed items. The .md files all use "*"s and not "-"s.

**Comment by MrCordeiro** [comment]
It's done!

**Pull request merged**